website_membership_random_order/models/membership_line.py

```diff
@@ -16,6 +16,7 @@ def search(self, args, offset=0, limit=None, order=None, count=False):
         won't be able to show all the records due to the pager and each time returning
         random results.
         """
+
```

Comment:
Why adding this empty line?

Reply:
I was debugging and I forgot to remove the line when I removed the debugger method. Changed.